src/lib/annotations/annotations.html

```diff
@@ -88,26 +88,77 @@
           this._parseElementAnnotations(node, list);
     },
 
-    _testEscape: function(value) {
-      var escape = value.slice(0, 2);
-      if (escape === '{{' || escape === '[[') {
-        return escape;
+    _bindingRegex: /([^{[]*)({{|\[\[)([^}\]]*)(?:]]|}})/g,

```

Comment:
Should we have some way to escape binding characters so they can be included in literal text?